app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int = 0,
+                 clean_mark: int = 0,
+                 brand: str = "") -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self, distance_from_city_center: float = 0,
+                 clean_power: int = 0,
+                 average_rating: float = 0,
+                 count_of_ratings: int = 0) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power and self.wash_single_car(car):
+                income += self.calculate_washing_price(car)
+                car.clean_mark = self.clean_power
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        comfort_class = car.comfort_class
+        clean_mark = car.clean_mark
+        if self.distance_from_city_center > 0:
+            return (comfort_class
+                    * ((self.clean_power - clean_mark)
+                       * self.average_rating) / self.distance_from_city_center)
```

Comment:
The `calculate_washing_price` method does not handle the case where `self.distance_from_city_center` is zero, which would lead to a division by zero error. Consider adding a check to handle this scenario.